deploy/operator/deploy.sh

```diff
@@ -30,4 +70,9 @@ OPENSHIFT_VERSIONS=$(echo ${OPENSHIFT_VERSIONS} |
                 rhcos_rootfs:  .value.rhcos_rootfs}
     }
     )')
-${__dir}/setup_assisted_operator.sh from_index_image
+
+if [ "${DISCONNECTED:-false}" == "true" ]; then
+    ${__dir}/setup_assisted_operator.sh from_community_operators
+else
+    ${__dir}/setup_assisted_operator.sh from_index_image
+fi
```

Comment:
for CI it's not true. you do want to use ``from_index_image`` either way
just let the user decide

Reply:
WDYT about
```bash
func="from_index_image"
if [ "${CI:-false}" = "false" ] && [ "${DISCONNECTED}" = "true" ]; then func="from_community_operators"; fi
```

Reply:
I prefer ``"${ASSISTED_DEPLOYMENT_METHOD:-from_community_operators}"``